lib/perl/Genome/Model/Command/Define/ImportedReferenceSequence.t

```diff
@@ -17,10 +17,11 @@ if (Genome::Sys->arch_os ne 'x86_64') {
     plan skip_all => 'requires 64-bit machine';
 }
 
+my $version = 1;
 my $cmd_class = 'Genome::Model::Command::Define::ImportedReferenceSequence';
 use_ok($cmd_class);
 
-my $data_dir = File::Temp::tempdir('ImportedAnnotationTest-XXXXX', CLEANUP => 1, TMPDIR => 1);
+my $data_dir = get_test_dir($cmd_class, $version);
```

Comment:
The trouble is that we need these to be network accessible and I agree that fewer external dependencies is better.  I think that I would change this to a `__FILE__.".d"`-type directory here.